CHANGES.md

```diff
@@ -38,6 +37,8 @@ To be released.
  -  Added .NET Core 2.2 as a targeted framework.  [[#209], [#561]]
  -  `TurnClient.AcceptRelayedStreamAsync()` became to ignore disconnected
     connections.  [[#469]]
+ -  `Swarm<T>.AppendBlocksAsync()` became to render blocks after all blocks
+    are appended.  [[#579]]
```

Comment:
Isn't this a bugfix?  Also, as `AppendBlockAsync()` method is `private`, we need to explain the behavior from the higher-level perspective.